aat/aat.js

```diff
@@ -0,0 +1,46 @@
+module.exports = {
```

Comment:
Will this require an authentication token or how does this work?

Reply:
Nope! It doesn't require an auth token & is currently working without one.

Reply:
Amazing!